pkg/manager/member/orphan_pods_cleaner.go

```diff
@@ -89,6 +110,23 @@ func (opc *orphanPodsCleaner) Clean(tc *v1alpha1.TidbCluster) (map[string]string
 			return skipReason, err
 		}
 
+		// check api server
```

Comment:
If PVC not found in cache, re-check from apiserver directly to make sure the PVC really not exist.

Reply:
done in https://github.com/pingcap/tidb-operator/pull/878/commits/7a3caf0fd6fe28106ffc5c67afad53bf438f51a1.